Added XStream files.

ReedOei opened the pull request:
Committed files as specified by the section "Files to commit to the repository" on https://sites.google.com/view/test-dependence-impact/guidelines-for-adding-a-new-subject

winglam commented on experiments/scripts/data/ORIG_MIN_DTs.txt:
```diff
@@ -1,9 +1,12 @@
+ActivemqCamel|1
 Ambari-Server|18
+Commons Lang|1
```
Why is this part of your changes?

winglam commented:
Can you provide a list of files that you modified, added, and deleted? Please separate the lists.

ReedOei commented:
It appears that none of the files in $DT_DATA/prioritization-dt-list, $DT_DATA/selection-dt-list, or $DT_DATA/parallelization-dt-list have been committed before, but they are on the list of files to commit on the guidelines. Should I remove the files that aren't relevant to XStream?

winglam commented:
Henry's pull request is moving the prioritization-dt-list and such to $DT_DATA. What do you mean to remove the files that aren't relevant to XStream? What files are you referring to?

ReedOei commented:
I mean removing the files such as selection-ActivemqCamel-auto-function-absolute.txt in the dt-list folders that don't have XStream in their name, because they aren't actually related to the task of committing XStream's files, especially if Henry already has a pull request to add them.

winglam commented:
Why would selection-ActivemqCamel-auto-function-absolute.txt have XStream in their name? Can you get me the list of files added, deleted, and modified?

winglam commented:
Your pull request here should be for XStream files, you should not even have Henry's changes for activemq stuff on your local repository. As you said though, please only commit XStream files for your pull request here.

ReedOei commented:
Here is a list of files added/modified, I didn't delete any files:

Deleted:
No files deleted.

Modified:
experiments/scripts/data/ORIG_MIN_DTs.txt
experiments/scripts/data/AUTO_MIN_DTs.txt
experiments/scripts/data/subject-data.csv
experiments/scripts/subj-constants.sh

Added:
experiments/scripts/subj-scripts/xstream.sh
All files in: experiments/xstream-1.2/
All files in: experiments/xstream-1.2.1/

experiments/scripts/data/selection-dt-list/
selection-XStream-auto-function-absolute.txt
selection-XStream-auto-function-original.txt
selection-XStream-auto-function-relative.txt
selection-XStream-auto-statement-absolute.txt
selection-XStream-auto-statement-original.txt
selection-XStream-auto-statement-relative.txt
selection-XStream-orig-function-absolute.txt
selection-XStream-orig-function-original.txt
selection-XStream-orig-function-relative.txt
selection-XStream-orig-statement-absolute.txt
selection-XStream-orig-statement-original.txt
selection-XStream-orig-statement-relative.txt

experiments/scripts/data/prioritization-dt-list/
prioritization-XStream-auto-function-absolute.txt
prioritization-XStream-auto-function-relative.txt
prioritization-XStream-auto-statement-absolute.txt
prioritization-XStream-auto-statement-relative.txt
prioritization-XStream-orig-function-absolute.txt
prioritization-XStream-orig-function-relative.txt
prioritization-XStream-orig-statement-absolute.txt
prioritization-XStream-orig-statement-relative.txt

experiments/scripts/data/parallelization-dt-list/
parallelization-XStream-auto-16-original.txt
parallelization-XStream-auto-16-time.txt
parallelization-XStream-auto-2-original.txt
parallelization-XStream-auto-2-time.txt
parallelization-XStream-auto-4-original.txt
parallelization-XStream-auto-4-time.txt
parallelization-XStream-auto-8-original.txt
parallelization-XStream-auto-8-time.txt
parallelization-XStream-orig-16-original.txt
parallelization-XStream-orig-16-time.txt
parallelization-XStream-orig-2-original.txt
parallelization-XStream-orig-2-time.txt
parallelization-XStream-orig-4-original.txt
parallelization-XStream-orig-4-time.txt
parallelization-XStream-orig-8-original.txt
parallelization-XStream-orig-8-time.txt

winglam commented on experiments/scripts/subj-constants.sh:
```diff
@@ -2,95 +2,111 @@
+  #${rootDir}/zeppelin-zengine-new/zeppelin-zengine/target
+  #${rootDir}/zeppelin-new/zeppelin-server/target
+  #/home/user/dependent-tests-impact/experiments/activemq-new/activemq-camel/target
+  /home/user/dependent-tests-impact/experiments/xstream-1.2.1/xstream/target
```
Can you use rootDir here? We will update the tool to automatically do this eventually.

winglam commented on experiments/scripts/subj-constants.sh:
```diff
@@ -2,95 +2,111 @@
+  #${rootDir}/ambari-new/ambari-server/target
+  #${rootDir}/zeppelin-zengine-new/zeppelin-zengine/target
+  #${rootDir}/zeppelin-new/zeppelin-server/target
+  #/home/user/dependent-tests-impact/experiments/activemq-new/activemq-camel/target
```
Why do you have activemq changes here still? Commit like this will cause merge conflicts with Henry. Neither of you should be modifying the contents of this file. You should only need to add to this file.

winglam commented on experiments/scripts/data/subject-data.csv:
```diff
@@ -6,4 +6,6 @@ Joda-Time,jodatime,jodatime-b609d7d66d/src/,jodatime-b609d7d66d/tests/org/,jodat
-Zeppelin-Zengine,zeppelin-zengine,zeppelin/zeppelin-zengine/src/main/,zeppelin/zeppelin-zengine/src/test/,zeppelin/zeppelin-zengine/target/,09e8f7f,380cfbf,zeppelin-zengine-new/zeppelin-zengine/src/main/,zeppelin-zengine-new/zeppelin-zengine/src/test/
\ No newline at end of file
+Zeppelin-Zengine,zeppelin-zengine,zeppelin/zeppelin-zengine/src/main/,zeppelin/zeppelin-zengine/src/test/,zeppelin/zeppelin-zengine/target/,09e8f7f,380cfbf,zeppelin-zengine-new/zeppelin-zengine/src/main/,zeppelin-zengine-new/zeppelin-zengine/src/test/
+ActivemqCamel,activemqCamel,activemq-old/activemq-camel/src/main/,activemq-old/activemq-camel/src/test/,activemq-old/activemq-camel/target/,abfe038,321ddd7,activemq-new/activemq-camel/src/main/,activemq-new/activemq-camel/src/test/
```
Please remove the activemq stuff here.

ReedOei commented:
I have made the requested changes.

winglam commented:
LGTM! Thanks for the changes.
